gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/DistributedRegion.java

```diff
@@ -1503,7 +1505,9 @@ private void getInitialImageAndRecovery(InputStream snapshotInputStream,
               RegionLogger.logPersistence(this.getFullPath(),
                   getDistributionManager().getDistributionManagerId(),
                   getPersistentID());
-              logger.info(LocalizedStrings.DistributedRegion_NEW_PERSISTENT_REGION_CREATED, new Object[] {this.getFullPath(), getPersistentID()});
+              if (!LocalRegion.isMetaTable(getFullPath())) {
+                logger.info(LocalizedStrings.DistributedRegion_NEW_PERSISTENT_REGION_CREATED, new Object[]{this.getFullPath(), getPersistentID()});
+              }
```

Comment:
Why are changes here required despite switching log-level to WARN in boot?

Going forward we should have selective turning on/off for these using log4j.